pages/events/index.vue

```diff
@@ -157,39 +134,95 @@ const extraFilters = [
   }
 ]
 
-const noEvents = (this as any).filteredEvents.length === 0
+const { data: upcomingEvents } = useLazyAsyncData(
+  'fetch-upcoming-events',
+  async () => await import('~/content/events/upcoming-community-events.json') as CommunityEvent[]
+)
 
-const getCheckedFilters = (filter: string) => (this as any)[filter]
+const { data: pastEvents } = useLazyAsyncData(
+  'fetch-past-events',
+  async () => await import('~/content/events/past-community-events.json') as CommunityEvent[]
+)
 
-const updateWholeFilter = (filter: string, filterValues: string[]) => {
-  const { commit } = this.$store
-  const payload = { filter, filterValues }
+const activeSet = ref('upcoming')
+const regionFilters = ref([] as string[])
```

Comment:
This type assertion can be replaced with a regular type.

```suggestion
const regionFilters = ref<string[]>([])
```

Reply:
I don't mind using one or the other, but right know we are mixing both types 😅 . I would be nice to choose one of them